src/lib/y2storage/proposal/settings_adjustment.rb

```diff
@@ -94,24 +96,28 @@ def regular_vol_description(mount_point, attrs)
         if attrs.include?(:proposed)
           # TRANSLATORS: %s is a mount point like "/home"
           _("do not propose a separate %s") % mount_point
-        elsif attrs.include?(:snapshots) && attrs.include?(:adjust_by_ram)
+        elsif attrs.sort == [:adjust_by_ram, :snapshots]
           # TRANSLATORS: %s is a mount point like "/home"
-          _("disabled snapshots and RAM-based size adjustments for %s") % mount_point
-        elsif attrs.include?(:snapshots)
+          _("disable snapshots and RAM-based size adjustments for %s") % mount_point
+        elsif attrs == [:snapshots]
           # TRANSLATORS: %s is a mount point like "/home"
-          _("disabled snapshots for %s") % mount_point
-        else
+          _("do not enable snapshots for %s") % mount_point
+        elsif attrs == [:adjust_by_ram]
           # TRANSLATORS: %s is a mount point like "/home"
           _("do not adjust size of %s based on RAM size") % mount_point
+        else
+          raise "Unknown volume adjustment"
```

Comment:
please include in exception also attrs, so we know what is there immediatelly. like "Unknown volume adjustment %{attrs.inspect}"